lib/editor/util/validation.js

```diff
@@ -233,12 +246,41 @@ export function validate (
         }
       }
       return false
+    case 'POSITIVE_INT':
+    case 'POSITIVE_NUM':
+      if (isRequiredButEmpty) {
```

Comment:
I'm not crazy about the validation logic for these two types being so closely linked. What do you think about moving the common "number validation" logic into a callable method that gets applied to the separate cases? For example, something like:

```
case 'POSITIVE_INT':
  const isInvalid = checkPositiveNumber(args)
  if (isInvalid) return isInvalid // if not a positive number, return validation issue immediately
  // Continue to check for integer validity.
  // ....
case 'POSITIVE_NUM':
  return checkPositiveNumber()
```

Reply:
Refactored as suggested.